chromeExtension/index.js

```diff
@@ -0,0 +1,26 @@
+"use strict";
+
+Object.defineProperty(exports, "__esModule", {
+  value: true,
+});
+
+exports.default = void 0;
+
+// Chrome Extension
+var _processDetailLoading = _interopRequireDefault(require("./process_detail_loading.json"));
```

Comment:
I dont think we need to do it like anymore since we did dependencies update in our frontends. I think its really ecmascript 6 vs ecmascript 5, but I dont remember exactly. You can just use same syntax as here https://github.com/Manifest-Labs/knowhow-animation/blob/master/app/dashboards/workflow/index.js

Reply:
https://github.com/Manifest-Labs/knowhow-animation/pull/56/commits/abb9f3f1f8baa2ee049a81531e2f790a01f530a5